### ISSUE TITLE
Overriding the "default" set in svg helper

### ISSUE THREAD

**Issue by xewl** [driesvints/blade-icons#66]
The helper (and incidentally the blade directive) don't seem to be able to see another icon set than `default`.

Proposition:
- Add the ability to override the default set key.

- Make a way to prefix in the first parameter of `svg($name)` which set to actually look in, overriding `default`.
eg. `@svg('other-set.subfolder.filename', ...)`

If this is already possible; I might be missing documentation on how to force a certain set to be used in this helper.

**Comment by driesvints** [comment]
> Add the ability to override the default set key.

It's already possible to remove the default set and replace it with another one. Is that what you mean?

> Make a way to prefix in the first parameter of svg($name) which set to actually look in, overriding default.
eg. @svg('other-set.subfolder.filename', ...)

This is also already possible. 

Please share code like your config file, the way you use the icons, etc. Are you using any of the other icon libraries?

**Comment by xewl** [comment]
_Long story short_, it's clear to be now, and working as I expect.😁✌

I'm guessing I ran into some issues with the name resolving.

The message threw me off; as I don't have a "default" set anymore.
`Svg by name "platform.event-canceled" from set "default" not found.`

My quote-on-quote default set, has the `icon-` prefix.
Another set was using the `icon-platfom` prefix.

I was doing `platform.` instead of `icon-platform-`
The latter also failed though, as it went looking into the first set with prefix `icon-` first.
Changed that prefix; resulting in `platform-event-canceled` now working.

Also needed to clear the config cache.. wasn't thinking about that;
because I'm using Apache inside WSL instead of `artisan serve`.
